info.yml

```diff
@@ -36,6 +36,22 @@ configuration:
         default_value_tk-maya: "{self}/general_actions.py:{self}/{engine_name}_actions.py"
         description: Hook which contains all methods for action management.
 
+    my_task_sort_fields:
+        type: list
+        description: "Controls what fields appear for sorting
+                     "
```

Comment:
```suggestion
        description: "Controls what fields appear for sorting"
```

Reply:
Actually I had intended to add more details there, that was just place holder. I'll update that thanks